userguide/tasks.rst

```diff
@@ -1670,6 +1670,10 @@ shows the configuration options for Cloud Syncs.
    |                     |                     | :guilabel:`Container`: Only appears when a :literal:`AZUREBLOB` credential is selected for the          |
    |                     |                     | :guilabel:`Credential`. Enter the name of the pre-configured Microsoft Azure Blob container.            |
    +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
+   | Upload Chunk Size   | integer             | Only appears with a *(B2)* :guilabel:`Credential`. Files are split into chunks of this size before      |
```

Comment:
As Warren would say: What it does first, then other information (like "Only appears with...") last.